src/analysis/classic/druid/balance/modules/Abilities.ts

```diff
@@ -0,0 +1,204 @@
+import SPELLS from 'common/SPELLS/classic/druid';
+import CoreAbilities from 'parser/core/modules/Abilities';
+import SPELL_CATEGORY from 'parser/core/SPELL_CATEGORY';
+
+class Abilities extends CoreAbilities {
+  spellbook() {
+    const combatant = this.selectedCombatant;
+    return [
+      // Rotational
+      {
+        spell: SPELLS.INSECT_SWARM.id,
+        category: SPELL_CATEGORY.ROTATIONAL,
+        gcd: { base: 1500 },
+      },
+      {
+        spell: [SPELLS.MOONFIRE.id, SPELLS.SUNFIRE.id],
+        category: SPELL_CATEGORY.ROTATIONAL,
+        gcd: { base: 1500 },
+      },
+      {
+        spell: SPELLS.STARFIRE.id,
+        category: SPELL_CATEGORY.ROTATIONAL,
+        gcd: { base: 1500 },
+      },
+      {
+        spell: SPELLS.STARSURGE.id,
+        category: SPELL_CATEGORY.ROTATIONAL,
+        gcd: { base: combatant.hasBuff(SPELLS.SHOOTING_STARS.id) ? 1000 : 1500 },
```

Comment:
this should be:

```suggestion
        gcd: { base: (combatant) => combatant.hasBuff(SPELLS.SHOOTING_STARS.id) ? 1000 : 1500 },
```

right? since otherwise you are basically just checking on pull for the buff

Reply:
Thought I could do a shortcut but wrote the module instead